### PR TITLE
Faster DB migration

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

**Review by rubickcz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -21,19 +21,44 @@ INSTALLED_APPS = [
...
+    },
+}
+```
+
+Snapshot and extensions are automatically loaded when database is preparing for tests too.
+
+### psqlclean
 
-PostgreSQL database requires often call `VACUUM` and `REINDEX`. The command `cleandb` performs these operations. Again you can specify database or root user with `--db-url` parameter.
+PostgreSQL database requires often call `VACUUM` and `REINDEX`. The command `cleandb` performs these operations.
```
Comment (Formulka) [inline README.md]: The command `psqlclean` performs these operations.

Comment (Formulka) [reply README.md]: I would also mention that the command vacuums all tables if no arguments are provided.

File: psycopg2_extension/management/commands/psqlclean.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        parser.add_argument(
+            '--tables-full',
+            dest='tables_full',
+            help='Tables to vaccum full separated with ,',
+            type=comma_separated_strings
+        )
+        parser.add_argument(
+            '--tables-reindex',
+            dest='tables_reindex',
+            help='Tables be excluded separated with ,',
```
Comment (Formulka) [inline psycopg2_extension/management/commands/psqlclean.py:25]: `Tables to reindex separated with ,`

File: psycopg2_extension/management/commands/psqlinit.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            help='Database extensions separated with ,',
+            type=comma_separated_strings
+        )
+        parser.add_argument(
+            '--noinput', '--no-input', action='store_false', dest='interactive',
+            help='Tells Django to NOT prompt the user for input of any kind.',
+        )
+        parser.add_argument(
+            '--nosnapshot', action='store_true', dest='no_snapshot',
+            help='Do not load snaopshot.',
```
Comment (Formulka) [inline psycopg2_extension/management/commands/psqlinit.py]: snapshot

File: psycopg2_extension/management/commands/psqlinit.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+    def handle(self, *args, **options):
+        connection = self._get_connection(options['database'])
+
+        if options['interactive']:
+            message = [
+                'This will delete existing database!\n',
+                'Are you sure you want to do this?\n\n'
+                "Type 'yes' to continue, or 'no' to cancel: "
+            ]
```
Comment (Formulka) [inline psycopg2_extension/management/commands/psqlinit.py]: why is there a comma at the first line but not the others?

Comment (matllubos) [reply psycopg2_extension/management/commands/psqlinit.py]: You are true, no comma isrequired here

**Pull request merged**